Documentation/Internal/ContainerLogV2-Linux.xml

```diff
@@ -8,9 +8,9 @@
       <IdentityComponent name="ClusterResourceId" envariable="AKS_RESOURCE_ID"/>
       <IdentityComponent name="ClusterRegion" envariable="AKS_REGION"/>
       <IdentityComponent name="Computer" envariable="HOSTNAME"/>
-      <IdentityComponent name="Tenant">TENANT_ID</IdentityComponent>
-      <IdentityComponent name="Role">ROLE_ID</IdentityComponent>
-      <IdentityComponent name="RoleInstance">ROLEINSTANCE_ID</IdentityComponent>
+      <IdentityComponent name="Tenant" envariable="AKS_RESOURCE_ID"/>
```

Comment:
why this is using the resource_id?  it already has <IdentityComponent name="ClusterResourceId" envariable="AKS_RESOURCE_ID"/>  either we do not need Tenant or use the right tenant id from AAD